tests/python/LookTest.py

```diff
@@ -1,28 +1,186 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright Contributors to the OpenColorIO Project.
 
-import unittest, os, sys
+import unittest
+import os
+import sys
+
 import PyOpenColorIO as OCIO
+from UnitTestUtils import TEST_NAMES, TEST_DESCS
+
 
 class LookTest(unittest.TestCase):
-    
-    def test_interface(self):
-        lk = OCIO.Look()
-        lk.setName("coollook")
-        self.assertEqual("coollook", lk.getName())
-        lk.setProcessSpace("somespace")
-        self.assertEqual("somespace", lk.getProcessSpace())
-        lk.setDescription("this is a test")
-        self.assertEqual("this is a test", lk.getDescription())
-        et = OCIO.ExponentTransform()
-        et.setValue([0.1, 0.2, 0.3, 0.4])
-        lk.setTransform(et)
-        oet = lk.getTransform()
-        vals = oet.getValue()
-        self.assertAlmostEqual(0.2, vals[1], delta=1e-8)
-        iet = OCIO.ExponentTransform()
-        iet.setValue([-0.1, -0.2, -0.3, -0.4])
-        lk.setInverseTransform(iet)
-        ioet = lk.getInverseTransform()
-        vals2 = ioet.getValue()
-        self.assertAlmostEqual(-0.2, vals2[1], delta=1e-8)
+    TEST_PROCESS_SPACES = ['raw', 'lnh', 'vd8', 'a.b.c.', '1-2-3-']
+
+    def setUp(self):
+        self.look = OCIO.Look()
+
+    def tearDown(self):
+        self.look = None
+
+    def test_name(self):
+        """
+        Test the setName() and getName() methods.
+        """
+
+        # Default initialized name value is ""
+        self.assertEqual(self.look.getName(), '')
+
+        for name in TEST_NAMES:
+            self.look.setName(name)
+            self.assertEqual(name, self.look.getName())
+
+        # Wrong type tests.
+        for invalid in (None, 1):
+            with self.assertRaises(TypeError):
+                self.look.setName(invalid)
+
+    def test_process(self):
```

Comment:
Recommend `test_process_space` instead of `test_process`

Reply:
Resolved in 2b67585.